modules/installation-creating-aws-subnet-localzone.adoc

```diff
@@ -35,42 +35,30 @@ requires:
 ----
 [
   {
-    "ParameterKey": "VpcId", <3>
-    "ParameterValue": "vpc-<random_string>" <4>
+    "ParameterKey": "VpcId",
+    "ParameterValue": "<value_of_VpcId>" <1>
   },
   {
-    "ParameterKey": "PublicRouteTableId", <5>
-    "ParameterValue": "<vpc_rtb_pub>" <6>
+    "ParameterKey": "PublicRouteTableId",
+    "ParameterValue": "<value_of_PublicRouteTableId>" <2>
   },
   {
-    "ParameterKey": "LocalZoneName", <7>
-    "ParameterValue": "<cluster_region_name>-<location_identifier>-<zone_identifier>" <8>
+    "ParameterKey": "LocalZoneName",
+    "ParameterValue": "<value_of_GroupName>" <3>
   },
   {
-    "ParameterKey": "LocalZoneNameShort", <9>
-    "ParameterValue": "<lz_zone_shortname>" <10>
-  },
-  {
-    "ParameterKey": "PublicSubnetCidr", <11>
-    "ParameterValue": "10.0.128.0/20" <12>
+    "ParameterKey": "PublicSubnetCidr",
+    "ParameterValue": "10.0.192.0/20" <4>
   }
 ]
 ----
-<1> Specify the cluster name that you used when you generated the `install-config.yaml` file for the cluster.
-<2> The VPC ID in which the Local Zone's subnet will be created.
-<3> Specify the `VpcId` value from the output of the CloudFormation template
+<1> Specify the VPC ID, output variable `VpcId` of the CloudFormation stack
 for the VPC.
-<4> The Public Route Table ID for the VPC.
-<5> Specify the `PublicRouteTableId` value from the output of the CloudFormation template for the VPC.
-<6>  The Local Zone name that the VPC belongs to.
-<7> Specify the Local Zone that you opted your AWS account into, such as `us-east-1-nyc-1a`.
-<8> The shortname of the AWS Local Zone that the VPC belongs to. This name must match the pattern `N[a-z]`.
-<9> Specify a short name for the AWS Local Zone that you opted your AWS account into, such as `<zone_group_identified><zone_identifier>`. For example, `us-east-1-nyc-1a` is shortened to `nyc-1a`.
-//How do we determine this shortname?
-<10> The CIDR block to allow access to the Local Zone.
-<11> Specify a CIDR block in the format `x.x.x.x/16-24`.
-//How do we know what this CIDR is?
-
+<2> Specify the Route Table ID, output variable `PublicRouteTableId` of the CloudFormation stack
+for the VPC.
+<3> Specify the AWS Local Zone name, output variable `ZoneName` of command `aws ec2 describe-availability-zones`
+executed on the section "Opting into AWS Local Zones".
```

Comment:
```suggestion
```suggestion
<3> Specify the AWS Local Zone name, which is the value of the `ZoneName` field in the `AvailabilityZones` object that you retrieve in the section "Opting into AWS Local Zones".
```
```